### ISSUE TITLE
Improved Usage documentation

### ISSUE THREAD

**Issue by gytic** [jorgerojas26/lazysql#117]
Hey,

I installed `Lazysql` today and tried connecting to a local SQLite DB.
It worked after some time of reading trough the shortcut info, but it wasn't the easiest thing to do.

I was wondering if it was possible to extend the `Usage` section in the `README.md` to give a (really) small tutorial
about how to get started. (Creating the first DB, connecting to an existing DB, executing a query, editing a row, filtering a row, switching tables) so the really basic stuff to get started, so someone freshly installing does not feel lost.

For me it was especially hard to figure out how to get back from the editor-screen to the tables (think in the end i just restarted) 
and switchting the table i am viewing.

I could try to write a simple getting started tutorial myself (I think I understand how the program works now much better) and submit a PR?

PS: I really love what you build here :)

**Comment by jorgerojas26** [comment]
That's a good suggestion, right now you have the `?` shortcut to show the keybindings. But, i know the docs could be better.

Another thing is that right now there is no way to create a table from the TUI, i will work on that soon.

Feel free to open a PR, we can get very biased as a power user and think it is easy enough to use. Your help is very welcome :)

Thank you.

**Comment by gytic** [comment]
Yeah, i also used the `?` to find my way trough the TUI, it was just a bit hard figuring out, which part of the UI was which part of the shortcut list etc, I think a small beginner tutorial can be very helpfull to get started there.

I will start working on something and open a PR in the next days.
